utils/scripts/ocm/ocm.py

```diff
@@ -844,6 +848,73 @@ def update_notification_email_address(self, addon_name, email_address, exit_on_f
             else:
                 return False
 
+    def get_latest_osd_candidate_version(self, osd_major_version, osd_minor_version):
+        """
+        get the latest  candidate version
+        Args:
+            osd_major_version (str):  Major version of the release. For example in 4.8.10 : 4 is major version
+            osd_minor_version (str):  Major version of the release. For example in 4.8.10 : 4 is minor version
+            Example 4.8 = osd_major_version.osd_minor_version       
+        """
+        cmd = ("ocm list versions --channel-group  candidate |"
+               " grep  ^{}.{}|tail -1".format(osd_major_version, osd_minor_version))
+        ret = execute_command(cmd)
+        if ret is None:
+            log.info("Failed  to get latest version of ODS ")
+        return ret
+
+    def get_all_osd_versions(self):
```

Comment:
is this function returning only Candindates?

Reply:
for now yes .. in the future we have a plan for stable also